src/images/tweet-arrow-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="37" height="37" viewBox="0 0 37 37">
```

Comment:
De her to svg filer ser ud til at være outdated, de burde ihvertfald ikke blive tilføjet igen. Kan du fjerne dem fra din branch? :)